miri-script/src/arg.rs

```diff
@@ -0,0 +1,112 @@
+use clap::{Parser, Subcommand};
+
+#[derive(Parser, Clone, Debug)]
+#[command(author, about, long_about = None)]
+pub struct Cli {
+    #[command(subcommand)]
+    pub commands: Subcommands,
+}
+
+#[derive(Subcommand, Clone, Debug)]
+pub enum Subcommands {
+    /// Installs the miri driver and cargo-miri.
+    /// Sets up the rpath such that the installed binary should work in any
+    /// working directory. Note that the binaries are placed in the `miri` toolchain
+    /// sysroot, to prevent conflicts with other toolchains.
+    Install {
+        /// Flags that are passed through to `cargo install`.
+        #[arg(trailing_var_arg = true)]
+        flags: Vec<String>,
```

Comment:
Is `String` the right type to use for a passed-through CLI argument? If the arguments are literally just passed through, I think `OsString` is better.